index.html

```diff
@@ -1049,7 +1049,7 @@ <h3 class="heading">CSS: External</h3>
 		  	</xmp>
 		  	<p class="explanationText">The external CSS file must be saved with <b>.css</b> extention, e.g.<b>"Style.css"</b>.Here's how our "Style.css" will look like..</p>
 		  	<xmp class="code">
-h3 {
+h1 {
```

Comment:
![screenshot from 2017-01-19 14-07-04](https://cloud.githubusercontent.com/assets/23294174/22099311/d6c7ef46-de50-11e6-996d-f37631d5d932.png)
because there is some typing error here